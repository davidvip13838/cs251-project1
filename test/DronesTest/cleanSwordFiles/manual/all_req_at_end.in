2 10 1
12
13
100000000
100000000
100000000
100000000
100000000
100000000
100000000
100000000
100000000
100000000
