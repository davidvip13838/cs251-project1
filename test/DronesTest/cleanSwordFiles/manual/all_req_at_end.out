100000000 0
100000000 0
100000001 1
100000002 2
100000003 3
100000004 4
100000005 5
100000006 6
100000007 7
100000008 8
